roles/tas_single_node/templates/manifests/rekor/backfill_redis.sh.j2

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
+LAST_INDEX_FILE="/tmp/rekor_last_index"
```

Comment:
Is the last index not queryable? Storing in a `/tmp` file is not great, because that will get wiped out on reboot. Can we either make this dynamic (query for it) or figure out a more reliable approach to store it (would a podman secret work? or at least a better file location somewhere in /var?)

Reply:
AFAIK, the last index filled in is not queryable, I will take a look at a better way to store it